csharp-tree-sitter.el

```diff
@@ -231,13 +233,13 @@
                   ))
     (indent-body . ;; if parent node is one of these and current node is in middle → indent
                  (block
-                  anonymous_object_creation_expression
-                  enum_member_declaration_list
-                  initializer_expression
-                  expression_statement
-                  declaration_list
-                  attribute_argument_list
-                  switch_body))
+                     anonymous_object_creation_expression
```

Comment:
What indentation rules are you using for this? It seems like you have some indentation settings to act on `block` as a macro, thus changing indentation? We don't need this, as it only will be reverted by me some time down the line as I adjust indentation. Please leave this as it is :)

Reply:
Oh, wait. This isn't macro? Then yes, I should revert this. Sorry for the changes.

Reply:
Hehe, no, but I can see why it looks like that. Maybe just put `block` as the second element of that list, to avoid this confusion down the line. No worries :)